index.html

```diff
@@ -124,8 +121,8 @@ <h2>
         Dependencies
       </h2>
       <p>
-        <code><dfn data-cite="ECMASCRIPT#sec-json.parse">JSON.parse</dfn></code> and
-        <code><dfn data-cite="ECMASCRIPT#sec-json.stringify">JSON.stringify</dfn></code> are
+        <code><dfn data-cite="ECMASCRIPT#sec-json.parse">JSON.parse()</dfn></code> and
+        <code><dfn data-cite="ECMASCRIPT#sec-json.stringify()">JSON.stringify()</dfn></code> are
```

Comment:
I think you need to avoid the parens on the link.

```suggestion
        <code><dfn data-cite="ECMASCRIPT#sec-json.parse">JSON.parse()</dfn></code> and
        <code><dfn data-cite="ECMASCRIPT#sec-json.stringify">JSON.stringify()</dfn></code> are
```

Reply:
oops! well spotted!